### PR TITLE
External Evaluation - Liberland

### PR DESCRIPTION
**Pull request by JosephKnecht-lab**
# External Evaluation 

Summary: Many of the deliverables were good. However, there are some missing items so the status is In Progress

### CODE REVIEW

**Comment by semuelle** [comment]
Thanks for the evaluation, @JosephKnecht-lab! @takahser will review it as soon as possible. We usually don't reject deliveries outright unless the grantee refuses to cooperate or violates T&Cs, so I'll leave this PR open so we can update the notes as we go.

File: evaluations/liberland_2_jkl.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Evaluation
+
+- **Status:** Rejected
```
Comment (takahser) [inline evaluations/liberland_2_jkl.md:3]:
```suggestion
- **Status:** In Progress
```

File: evaluations/liberland_2_jkl.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+2023-04-14 13:26:06 ⛓  Native runtime: Liberland-8 (liberland-node-0.tx1.au10)    
+2023-04-14 13:26:09 [0] 💸 generated 1 npos voters, 1 from validators and 0 nominators    
+2023-04-14 13:26:09 [0] 💸 generated 1 npos targets    
+2023-04-14 13:26:10 You're running on a system with a broken `madvise(MADV_DONTNEED)` implementation. This will result in lower performance.    
+2023-04-14 13:26:27 🔨 Initializing Genesis block/state (state: 0x12f5…4c6f, header-hash: 0x9f82…9441)    
+2023-04-14 13:26:27 👴 Loading GRANDPA authority set from genesis on what appears to be first startup. 
+```
+
+###  Concluding Remarks
+
```
Comment (takahser) [inline evaluations/liberland_2_jkl.md:47]:
```suggestion
#### Remarks by @JosephKnecht-lab
```

File: evaluations/liberland_2_jkl.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+Based on my evaluation, you have not provided all the deliverables as described in you application. Thus I have to reject this delivery. My advise is to update your initial application with the exact deliverables that you are delivering for this milestone, before it can be accepted. 
+
+Furthermore, a lot of your pallets are derivative work from already existing pallets (such as registry, nfts) and I would expect more original work for grant submission. 
+
+Note that this evaluation is not binding since I do not work of Web3 Foundation, so the Web3 team can accept your delivery in its current state.
+
```
Comment (takahser) [inline evaluations/liberland_2_jkl.md:53]:
```suggestion
#### Remarks by @takahser

(TODO)

```

**Review by takahser — APPROVED** [review]
@JosephKnecht-lab thanks for your evaluation, it's very much appreciated.
I've changed the base to a dedicated branch for this evaluation so I'm going to merge this PR now. Note that I'm still going to update the evaluation while considering the points you've made. Also, I'm changing the status from `Rejected` to `In Progress` due to the reasons [mentioned](https://github.com/w3f/Grant-Milestone-Delivery/pull/829#issuecomment-1523128013) by @semuelle.

We'll transfer your KSM reward to the address you provided in the evaluation: `Ha27MB4gKH36ieRPa1T1ASqhUz4sVxMoVmK7o5CS6tXDqjW`. Could you just quickly confirm that it is correct?

**Comment by RouvenP** [comment]
hi @stojanov-igor we transferred the KSM yesterday

**Pull request merged**